packages/flutter_tools/test/commands/create_test.dart

```diff
@@ -42,7 +42,11 @@ void main() {
     });
 
     tearDown(() {
-      temp.deleteSync(recursive: true);
+      try {
+        temp.deleteSync(recursive: true);
+      } catch (e) {
```

Comment:
we should call out the specific errors we want to ignore. We don't want to silence NoSuchMethodErrors or whatnot.

Reply:
👍 done